DESCRIPTION

```diff
@@ -5,6 +5,7 @@ Authors@R: c(
     person("Max", "Kuhn", , "max@rstudio.com", role = c("aut", "cre"),
            comment = c(ORCID = "0000-0003-2402-136X")),
     person("Daniel", "Falbel", , "daniel@rstudio.com", role = "aut"),
+    person("Hamada S.", "Badr", , "badr@jhu.edu", role = "ctb"),
```

Comment:
We generally do not add contributors to the description file (because it can get huge; see `broom`). If you would like something formal, you can create a `contributors.md` file and/or add a note about the change to the `news.md file`

Reply:
No problem. Some maintainers request it, but I understand the size of contributions here. Just reverted it. Thanks for all your efforts; it's greatly appreciated.